### DISCUSSION TITLE
Cannot believe this works so well!

### DISCUSSION THREAD

**Discussion by pblack476** [AntiMicroX/antimicrox#405] [General]
I actually got a PDP Rock Candy Switch Controller to work on linux with this by configuring my emulator to keyboard input and the keyboard itself to the controller. I was ready to dual boot windows for this. Thanks a lot! It works **seamlessly**

**Comment by pktiuk** [comment]
I'm glad to hear that.

**Reply by tplaiho777** [reply]
I am really happy too with AntiMicroX. It saved me lots of money cos I do not need to reject my old games for something else, which supports my Cobra 5 joystick combo. I could help to translate it into Finnish if in a need. Very nice software.
